docs/content/basics.md

```diff
@@ -1,334 +1,369 @@
-```
-
-You should now have two routes for your application:
-
-```
-$ flynn route
-ROUTE                             SERVICE      ID                                         STICKY  LEADER  PATH
-http:example.com                  example-web  http/74b05faf-c062-42f2-8ffe-678cfa3c061b  false   false   /
-http:example.demo.localflynn.com  example-web  http/2e37467e-08fc-47e5-853b-4f0574cb6871  false   false   /
-```
-
-HTTP requests to `example.com` should be routed to the web processes:
-
-```
-$ curl http://example.com
-Hello from Flynn on port 8080 from container 8f61a0f9-0582-474c-a996-1bec7d496f2a
-Hits = 6
-```
-
-You could now modify your application to respond differently based on the HTTP Host
-header (which here could be either `example.demo.localflynn.com` or `example.com`).
-
-## Multiple Processes
-
-So far the example application has only had one process type (i.e. the `web` process),
-but applications can have multiple process types which can be scaled individually.
-
-Lets add a simple command that will print the time every second.
-
-Make a new directory called `clock`, and add the following to `clock/main.go`:
-
-```go
-package main
-
-import (
-	"fmt"
-	"time"
-)
-
-func main() {
-	for t := range time.NewTicker(time.Second).C {
-		fmt.Println(t)
-	}
-}
-```
-
-Create the `clock` process type by adding the following to `Procfile`:
-
-```
-clock: clock
-```
-
-Release these changes:
-
-```
-$ git add clock/ Procfile
-$ git commit -m "Add clock service"
-$ git push flynn master
-```
-
-Scale the `clock` service to one process and get its output:
-
-```
-$ flynn scale clock=1
-
-$ flynn ps
-ID                                          TYPE  STATE   CREATED        RELEASE
-flynn-5a2b2364-cfb6-411e-86f8-af9298994f09  clock  up     13 seconds ago  7f81b96d-3834-4eed-8b05-9ce66cc07b54
-flynn-c0b01f0f-4236-4d01-94eb-fe8c16f3dc0e  web    up     13 seconds ago  7f81b96d-3834-4eed-8b05-9ce66cc07b54
-flynn-7453bc70-6b79-4776-9f32-c57506ba28f6  web    up     13 seconds ago  7f81b96d-3834-4eed-8b05-9ce66cc07b54
-flynn-4fc38d7d-06d8-4285-a225-ae0cdeb58e03  web    up     13 seconds ago  7f81b96d-3834-4eed-8b05-9ce66cc07b5
-
-$ flynn log -t clock
-2016-07-26T13:47:11.217109Z app[clock.flynn-5a2b2364-cfb6-411e-86f8-af9298994f09]: 2016-07-26 13:47:11.216175147 +0000 UTC
-2016-07-26T13:47:12.221625Z app[clock.flynn-5a2b2364-cfb6-411e-86f8-af9298994f09]: 2016-07-26 13:47:12.221232786 +0000 UTC
-2016-07-26T13:47:13.217002Z app[clock.flynn-5a2b2364-cfb6-411e-86f8-af9298994f09]: 2016-07-26 13:47:13.21673476 +0000 UTC
-2016-07-26T13:47:14.216574Z app[clock.flynn-5a2b2364-cfb6-411e-86f8-af9298994f09]: 2016-07-26 13:47:14.216298339 +0000 UTC
-2016-07-26T13:47:15.216373Z app[clock.flynn-5a2b2364-cfb6-411e-86f8-af9298994f09]: 2016-07-26 13:47:15.21610766 +0000 UTC
-```
-
-## Run
-
-An interactive one-off process may be spawned in a new container:
-
-```
-$ flynn run bash
-```
-
-*See [here](/docs/cli#run) for more information on the `flynn run` command.*
+---

```

Comment:
@imjorge any idea why this diff is the whole file? Did you edit on Windows (maybe you switched all the `LF` for `CRLF`)?